crypto/nrf_cc310_platform/include/nrf_cc3xx_platform.h

```diff
@@ -35,13 +35,21 @@ extern "C"
 {
 #endif
 
-/**@brief Function to initialize the nrf_cc3xx_platform with rng support
+/**@brief Function to initialize the nrf_cc3xx_platform with rng support.
+ *        The function is using CTR_DRBG to generate a random seed.
  *
  * @return Zero on success, otherwise a non-zero error code.
  */
 int nrf_cc3xx_platform_init(void);
 
 
+/**@brief Function to initialize the nrf_cc3xx_platform with rng.
+ *        The function is using HMAC_DRBG to generate a random seed.
+ *
+ * @return Zero on success, otherwise a non-zero error code.
+ */
+int nrf_cc3xx_platform_init_hmac_drbg(void);
```

Comment:
Suggestion: It's not clear if this can be used after `nrf_cc3xx_platform_init` has been used.

I.e. if they are mutually exclusive or not.